### ISSUE TITLE
Can't change shortcut for toggling terminal to cmd+` instead of ctrl+`

### ISSUE THREAD

**Issue by sunnypatel** [Microsoft/vscode#17144]
- VSCode Version: Code 1.7.2 (7ba55c5860b152d999dda59393ca3ebeb1b5c85f, 2016-11-21T22:01:10.756Z)
- OS Version: Darwin x64 15.6.0
- Extensions:

|Extension|Author|Version|
|---|---|---|
|cjsx|crisward|0.0.6|
|git-common-commands-suite|samschneller|0.2.2|
|vscode-jumpy|wmaurer|0.3.1|

---

Steps to Reproduce:

1.
2.

**Comment by kieferrm** [comment]
Even when disabling the macOS shortcut for focussing the next window, the shortcut does not work when defined in `keybindings.json`. Interesting when using `Add Keybinding` inside the editor and you type `` Cmd+` `` you get `Cmd+Unknown` in the dialog.

**Comment by alexandrudima** [comment]
@kieferrm What keyboard layout are you using? When seeing `cmd+unknown` it usually means we do not receive this keybinding in vscode. There is a glitch in that widget where pressing only `cmd` is rendered as `cmd+unknown` due to how the Keybinding represents a modifier key. I've filed #17158 to track that.

@kieferrm Does pressing `` cmd+` `` work for you in Chrome when visiting https://www.w3.org/2002/09/tests/keys.html

More info available at: https://github.com/Microsoft/vscode/wiki/Troubleshooting-Keybindings

**Comment by kieferrm** [comment]
@alexandrudima This is what I get in Chrome on the [test page](https://www.w3.org/2002/09/tests/keys.html) when pressing `` Cmd+` ``:

> keydown (0): [keyCode: 91][shift: false][ctrl: false][alt: false][meta: true]
> keydown (1): [keyCode: 91][shift: false][ctrl: false][alt: false][meta: true]
> keyup (2): [keyCode: 91][shift: false][ctrl: false][alt: false][meta: false]
> keyup (3): [keyCode: 91][shift: false][ctrl: false][alt: false][meta: false]

I use the regular US keyboard layout:
<img width="668" alt="screen shot 2016-12-14 at 6 10 55 am" src="https://cloud.githubusercontent.com/assets/4674940/21185100/2a6bb230-c1c4-11e6-9af9-6b53fa17394b.png">

**Comment by alexandrudima** [comment]
I have just tried today on the mac mini in the office.

I have followed some steps explained in http://apple.stackexchange.com/a/50759 and proceeded to uncheck the following checkbox:

![screen shot 2017-03-24 at 16 15 36](https://cloud.githubusercontent.com/assets/5047891/24300556/25fb0024-10ad-11e7-831e-22ad41bd378a.png)

I then switched to VSCode and used the define keybinding widget. Still, `` cmd+` `` **does not reach VSCode**. 

There is no possible code change I can do in VS Code if we don't get a `keydown` event. I am no mac expert to even begin figuring this out.
